tests/templates/service/check_render.py

```diff
@@ -41,7 +41,7 @@ def check_file_template(self):
 
 class TemplateRenderingService(Service):
     NO_VARIABLE = "fixed content"
-    SIMPLE_VARIABLE = "Hello {{name}}!"
+    SIMPLE_VARIABLE = "Hello {{a_field}}!"
```

Comment:
This change made me realize we might be using `name` as a field in services elsewhere. If we make this change I see at least `connect.py` in kafkatest will need modifications.